docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -103,6 +103,14 @@ children:
       url: Parsing-requests.html
       output: 'web, pdf'
 
+  - title: 'Middleware'
+    url: Middleware.html
+    output: 'web, pdf'
```

Comment:
I feel this page does not belong to [Key Concepts](https://loopback.io/doc/en/lb4/Concepts.html) and should be moved to [Behind the scene](https://loopback.io/doc/en/lb4/Behind-the-scene.html).

I am expecting that new LB users won't need to understand the information from this page until they get into really advanced use cases.

Reply:
I put it under `Key Concepts` for two reasons:

- It's next to `Interceptors`
- Application developers need to add middleware